core/src/main/java/projectzero/UMLClass.java

```diff
@@ -2,12 +2,18 @@
 
 public class UMLClass {
     private String name;
-    public UMLClass(String name){
+
+    public UMLClass() {
+    }
```

Comment:
The Jackson ObjectMapper requires the object have a default constructor. Jackson requires default constructors, along with getters and setters, because that is how they interface with POJOs. Other third party libraries do the same.